package/MDAnalysis/coordinates/timestep.pyx

```diff
@@ -493,20 +493,31 @@ cdef class Timestep:
         vectors lengths followed by their respective angle, or as three
         triclinic vectors.
 
-          >>> ts.dimensions
-          array([ 13.,  14.,  15.,  90.,  90.,  90.], dtype=float32)
-          >>> ts.triclinic_dimensions
-          array([[ 13.,   0.,   0.],
-                 [  0.,  14.,   0.],
-                 [  0.,   0.,  15.]], dtype=float32)
+          .. testsetup::
+
+          >>> import MDAnalysis as mda
+          >>> from MDAnalysis.tests.datafiles import TPR, XTC
+          >>> import numpy as np
+          >>> u = mda.Universe(TPR, XTC)
+          >>> ts = u.trajectory[0] 
```

Comment:
Based on a similar PR https://github.com/MDAnalysis/mdanalysis/pull/4374/files these lines should be indented to look like
```
         .. testsetup::
            import MDAnalysis as mda
            from MDAnalysis.tests.datafiles import TPR, XTC
            ...
```
and I am not 100% sure if `>>>` should remain.